Kitodo/src/test/java/org/kitodo/production/metadata/MetadataEditorIT.java

```diff
@@ -77,22 +86,25 @@ public static void cleanDatabase() throws Exception {
 
     @Test
     public void shouldAddLink() throws Exception {
-        File metaXmlFile = new File("src/test/resources/metadata/4/meta.xml");
+        int parentId = testProcessIds.get(MockDatabase.HIERARCHY_PARENT);
+        int childId = testProcessIds.get(MockDatabase.HIERARCHY_CHILD_TO_ADD);
+        File metaXmlFile = new File("src/test/resources/metadata/" + parentId + "/meta.xml");
         List<String> metaXmlContentBefore = FileUtils.readLines(metaXmlFile, StandardCharsets.UTF_8);
 
-        MetadataEditor.addLink(ServiceManager.getProcessService().getById(4), "0", 7);
+        MetadataEditor.addLink(ServiceManager.getProcessService().getById(parentId), "0", childId);
 
         assertTrue("The link was not added correctly!",
-            isInternalMetsLink(FileUtils.readLines(metaXmlFile, StandardCharsets.UTF_8).get(37), 7));
+            isInternalMetsLink(FileUtils.readLines(metaXmlFile, StandardCharsets.UTF_8).get(36), childId));
 
         FileUtils.writeLines(metaXmlFile, StandardCharsets.UTF_8.toString(), metaXmlContentBefore);
-        FileUtils.deleteQuietly(new File("src/test/resources/metadata/4/meta.xml.1"));
+        FileUtils.deleteQuietly(new File("src/test/resources/metadata/" + parentId + "/meta.xml.1"));
     }
 
     @Test
     public void shouldAddMultipleStructuresWithoutMetadata() throws Exception {
-
-        File metaXmlFile = new File("src/test/resources/metadata/2/meta.xml");
+        int testProcessId = MockDatabase.insertTestProcess(TEST_PROCESS_TITLE, 1, 1, 1);
+        ProcessTestUtils.copyTestMetadataFile(testProcessId, TEST_METADATA_FILE);
+        File metaXmlFile = new File("src/test/resources/metadata/" + testProcessId + "/meta.xml");
```

Comment:
It would be good if the path or parts of the path could be generalised and reused.